modules/nf-core/plink/genome/main.nf

```diff
@@ -0,0 +1,39 @@
+process PLINK_GENOME {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/plink:1.90b6.21--h031d066_5':
+        'biocontainers/plink:1.90b6.21--h031d066_5' }"
+
+    input:
+    tuple val(meta), path(bed), path(bim), path(fam)
+
+    output:
+    tuple val(meta), path("*.genome")   , emit: genome
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    if( "$bed" == "${prefix}.bed" ) error "Input and output names are the same, use \"task.ext.prefix\" in modules.config to disambiguate!"
+
+    """
+    plink \\
+        --bfile ${bed.baseName} \\
```

Comment:
```suggestion
        --bed ${bed} \\
        --bim ${bim} \\
        --fam ${fam} \\
```
Better to do it explicitly. Names could be inconsistent across the files.

Reply:
Thank you, I've fixed this